sensor/kubernetes/enforcer/cronjob/suspend.go

```diff
@@ -0,0 +1,50 @@
+package cronjob
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/branding"
+	kubernetesPkg "github.com/stackrox/rox/pkg/kubernetes"
+	"github.com/stackrox/rox/pkg/retry"
+	"github.com/stackrox/rox/sensor/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Suspend suspends the cron job
+func Suspend(ctx context.Context, client kubernetes.Interface, deploymentInfo *central.DeploymentEnforcement) (err error) {
+	depName := deploymentInfo.GetDeploymentName()
+	depType := deploymentInfo.GetDeploymentType()
+
+	patch := "{\"spec\" : {\"suspend\" : true }}"
+	if ok, apiErr := utils.HasAPI(client, "batch/v1", kubernetesPkg.CronJob); ok && apiErr != nil {
```

Comment:
Looks like this should be
```suggestion
	if ok, apiErr := utils.HasAPI(client, "batch/v1", kubernetesPkg.CronJob); ok && apiErr == nil {
```